src/plugins/intel_cpu/src/compiled_model.cpp

```diff
@@ -342,5 +342,13 @@ void CompiledModel::export_model(std::ostream& modelStream) const {
     serializer << m_model;
 }
 
+void CompiledModel::release_memory() {
+    for (auto&& graph : m_graphs) {
+        GraphGuard::Lock graph_lock{graph};
+        auto ctx = graph_lock._graph.getGraphContext();
+        ctx->getNetworkMemoryControl()->releaseMemory();
```

Comment:
can we release memory only for graphs which don't have running inference requests?

Reply:
In that case we would have to add a delayed release for the others. I'm not sure we are really want to have nonuniform memory release across streams.